usersimcrs/items/ratings.py

```diff
@@ -112,3 +115,23 @@ def get_random_user_id(self) -> str:
             User ID.
         """
         return random.choice(list(self._user_ratings.keys()))
+
+    def create_split(
+        self, historical_ratio: float
+    ) -> Tuple["Ratings", "Ratings"]:
```

Comment:
```suggestion
    ) -> Tuple[Ratings, Ratings]:
```

Reply:
That didn't work for me, as we're inside Ratings, so that type doesn't exist yet. This is the simplest known workaround.